### PR TITLE
Feat/using bootstrap variables

### LINKED ISSUE #181: Grey colour sync
### Design decision 2020-03-23
 
**What is this**
Grey shade synk meeting with Design Management and Bootstrap (Robert Hjalmers)

**Change**
- Design Library and Sketch will use the same naming convention as SEB Bootstrap (from "grey 8" to "grey 800")
- Design Library and Sketch will change colours 1-7 and use same grey shades as SEB Bootstrap
- Bootstrap will change the two darker colours 800 and 900 (800: from #343434 to #333333, 900: from #212121 to #1A1A1A)

**Correct grey shades**
$white:    #fff !default;
$gray-100: #f8f8f8 !default;
$gray-200: #e9e9e9 !default;
$gray-300: #dedede !default;
$gray-400: #cecece !default;
$gray-500: #adadad !default;
$gray-600: #868686 !default;
$gray-700: #494949 !default;
$gray-800: #333333 !default;
$gray-900: #1A1A1A !default;
$black:    #000 !default; 

**Reason**
Our designers and developers **must** have the same colour references.

If this issue is not relevant to you, please see it as information. Feel free to comment and close it.

Thank you!
/Ulrika

### PR DESCRIPTION
**Pull request by yousifalraheem**
- Using bootstrap variables instead of defining them in a file and risk being outdated (fixes #181)
- Using @sebgroup/frontend-tools instead of defining them in utils folder
- Also includes bug fixes and improvements
- Generates missing docs pages

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -1,25 +1 @@
...
-    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
-</head>
-
-<body>
-    <div id="app"></div>
-<script type="text/javascript" src="app.js"></script></body>
-
-</html>
\ No newline at end of file
+<!doctype html><html lang="en"><head><title>React Components Docs</title><base href="https://sebgroup.github.io/react-components/"><meta charset="UTF-8"><meta name="format-detection" content="telephone=no"><meta name="msapplication-tap-highlight" content="no"><meta name="mobile-web-app-capable" content="yes"><meta name="apple-mobile-web-app-capable" content="yes"><meta name="apple-mobile-web-app-status-bar-style" content="translucent-black"><meta name="apple-mobile-web-app-title" content="BaseApp"><meta http-equiv="cleartype" content="on"><meta name="viewport" content="user-scalable=no,initial-scale=1,maximum-scale=1,minimum-scale=1,width=device-width"><link rel="shortcut icon" href="favicon.ico" type="image/x-icon"></head><body><div id="app"></div><script src="app.js"></script></body></html>
```
Comment (mario-subo) [inline docs/index.html:1]: is this one line because of prettier? I think it should ignore html files.

File: src/Slider/slider-style.scss [hunk 2]
```diff
@@ -306,16 +301,10 @@ $disabled-thumb-border-color: #8c8c8c;
                 }
                 &.with-transitions {
                     .custom-slider-slider-after {
-                        @include transition((width $transition-time ease-out, transform $transition-time linear));
+                        /** @include transition((width $transition-time ease-out, transform $transition-time linear)); */
```
Comment (mario-subo) [inline src/Slider/slider-style.scss:304]: why is this commented out?

File: src/Slider/slider-style.scss [hunk 3]
```diff
@@ -306,16 +301,10 @@ $disabled-thumb-border-color: #8c8c8c;
...
                     }
                     .custom-slider-thumb {
-                        @include transition(
-                            (
-                                left $transition-time ease-out,
-                                transform $transition-time ease-in-out,
-                                box-shadow $transition-time ease-in
-                            )
-                        );
+                        /** @include transition((left $transition-time ease-out, transform $transition-time ease-in-out, box-shadow $transition-time ease-in)); */
```
Comment (mario-subo) [inline src/Slider/slider-style.scss:307]: why comment out? remove if not needed

File: src/Accordion/accordion-style.scss [hunk 4]
```diff
@@ -1,6 +1,5 @@
-$blue-dark: #0092e1;
-$white: #fff;
-$gray-8: #e6e6e6 !default;
+@import "../../node_modules/@sebgroup/bootstrap/scss/variables";
```
Comment (omarVodiaK) [inline src/Accordion/accordion-style.scss:1]: There most be a way to create alias for variables file
https://medium.com/front-end-weekly/creating-alias-for-package-imports-in-react-99d455284029

File: src/Dialogue/dialogue-style.scss [hunk 5]
```diff
@@ -75,7 +59,7 @@ $close-button-size: 15px;
                     display: inline-block;
                     .dialogue-button {
                         min-width: 120px;
-                        font-size: $default;
+                        font-size: $font-size-sm;
```
Comment (omarVodiaK) [inline src/Dialogue/dialogue-style.scss:62]:
```suggestion
                        font-size: $font-size-base;
```

File: src/ImageCropper/image-cropper-style.scss [hunk 6]
```diff
@@ -1,30 +1,12 @@
...
+
 $blue-darkest: #05578c;
-$white: #fff;
-$transparent: transparent;
-$gray-300: #dedede;
 $backdrop-background: rgba(7, 6, 6, 0.7);
-$cropper-dialogue-background: #FFF;
-$material-box-shadow: 0 7px 28px rgba(0, 0, 0, 0.25),
-0 2px 2px rgba(0, 0, 0, 0.22);
+$cropper-dialogue-background: $white;
```
Comment (omarVodiaK) [inline src/ImageCropper/image-cropper-style.scss:6]: i dont believe this is required, $white is already a variable

File: src/Stepper/stepper-style.scss [hunk 7]
```diff
@@ -100,14 +86,14 @@ $stepper-font-size: 24px;
...
         &.alert-danger {
-            border-color: #F03529;
-            color: #F03529;
+            border-color: #f03529;
+            color: #f03529;
         }
         &.alert-warning {
-            border-color: #F8A000;
-            color: #F8A000;
+            border-color: #f8a000;
```
Comment (omarVodiaK) [inline src/Stepper/stepper-style.scss:95]: can we move this to a variable ?

**Pull request merged**